usr/lib/api/apiutil.c

```diff
@@ -58,7 +64,7 @@ CK_RV CreateProcLock(void)
         /* The slot mgr daemon should have already created lock,
          * so just open it so we can get a lock...
          */
-        xplfd = open(OCK_API_LOCK_FILE, O_RDONLY);
+        xplfd = open(OCK_API_LOCK_FILE, OPEN_MODE);
```

Comment:
The commit message still mentions that Linux would also need O_WRONLY, which is no longer true. Please adjust accordingly.

Reply:
The commit message still reads _... the file locking featureset from BSD (`flock`)
requires the file to be opened in write mode on AIX, but Linux does not care
about the mode in which the file is opened. Therefore, the minority wins and
the default mode is now changed to be the one that AIX needs._
I guess this is not entirely true anymore, at least in regards to Linux. Linux keeps using O_RDONLY, so maybe this needs a rephrase.

Reply:
Ah, I missed this one. Changed it in the latest changeset.